packages/agoric-cli/src/commands/vaults.js

```diff
@@ -3,23 +3,41 @@
 // @ts-check
 /* eslint-disable func-names */
 /* global fetch, process */
-import { Command } from 'commander';
+import { Command, Option } from 'commander';
 import { normalizeAddressWithOptions } from '../lib/chain.js';
-import { makeRpcUtils } from '../lib/rpc.js';
+import { makeRpcUtils, storageHelper } from '../lib/rpc.js';
 import {
   lookupOfferIdForVault,
   makeAdjustSpendAction,
   makeCloseSpendAction,
   makeOpenSpendAction,
 } from '../lib/vaults.js';
 import { getCurrent, outputAction } from '../lib/wallet.js';
+import { asPercent, makeAmountFormatter } from '../lib/format.js';
 
 /** @typedef {import('@agoric/smart-wallet/src/offers').OfferSpec} OfferSpec */
 /** @typedef {import('@agoric/smart-wallet/src/offers').OfferStatus} OfferStatus */
 /** @typedef {import('@agoric/smart-wallet/src/smartWallet').BridgeAction} BridgeAction */
 
 const { vstorage, fromBoard, agoricNames } = await makeRpcUtils({ fetch });
 
+/**
+ * @param {number} manager
+ */
+const getGovernanceState = async manager => {
```

Comment:
not to be confused with factory governance:
```suggestion
const getManagerGovernanceState = async manager => {
```